packages/honeycomb-opentelemetry-web/src/global-errors-autoinstrumentation.ts

```diff
@@ -26,6 +27,34 @@ export class GlobalErrorsInstrumentation extends InstrumentationAbstract {
     this._isEnabled = enabled;
   }
 
+  _computeStackTrace = (error: Error | undefined) => {
+    if (!error) {
+      return {};
+    }
+
+    // OTLP does not accept arrays of objects
+    // breaking down the stack into arrays of strings/numbers
+    const structuredStack: StackFrame[] = computeStackTrace(error).stack;
```

Comment:
Is there a file key passed on the structured stack trace?

Reply:
Did a quick check and `TraceKit` does not give a key called file. I'm thinking their file key is the `url` key we currently have here.